apache-hbase-mixin/dashboards/apache-hbase-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1032 @@
+        },
+        properties: [
+          {
+            id: 'noValue',
+            value: 'false',
+          },
+        ],
+      },
+    ],
+  },
+  options: {
+    cellHeight: 'md',
+    footer: {
+      countRows: false,
+      fields: '',
+      reducer: [
+        'sum',
+      ],
+      show: false,
+    },
+    showHeader: true,
+    sortBy: [],
+  },
+  pluginVersion: '10.2.0-61719',
+  transformations: [
+    {
+      id: 'merge',
+      options: {},
+    },
+    {
+      id: 'organize',
+      options: {
+        excludeByName: {
+          Time: true,
+          'Time 1': true,
+          'Time 2': true,
+          Value: true,
+          'Value #A': true,
+          'Value #B': true,
+          __name__: true,
+          '__name__ 1': true,
+          '__name__ 2': true,
+          clusterid: true,
+          'clusterid 1': true,
+          'clusterid 2': true,
+          context: true,
+          'context 1': true,
+          'context 2': true,
+          hbase_cluster: true,
+          'hbase_cluster 1': true,
+          'hbase_cluster 2': true,
+          instance: false,
+          'instance 1': false,
+          'instance 2': true,
+          isactivemaster: false,
+          'isactivemaster 1': false,
+          'isactivemaster 2': true,
+          job: true,
+          'job 1': true,
+          'job 2': true,
+          liveregionservers: true,
+          'liveregionservers 1': true,
+          'liveregionservers 2': true,
+          servername: false,
+          'servername 1': false,
+          'servername 2': true,
+          zookeeperquorum: true,
+          'zookeeperquorum 1': true,
+          'zookeeperquorum 2': true,
+        },
+        indexByName: {
+          Time: 4,
+          'Value #A': 12,
+          'Value #B': 13,
+          __name__: 5,
+          clusterid: 7,
+          context: 8,
+          hbase_cluster: 6,
+          hostname: 1,
+          instance: 2,
+          isactivemaster: 3,
+          job: 9,
+          liveregionservers: 10,
+          servername: 0,
+          zookeeperquorum: 11,
+        },
+        renameByName: {
+          Time: '',
+          deadregionservers: 'Dead server name',
+          hostname: 'Hostname',
+          instance: 'Instance',
+          'instance 1': '',
+          isactivemaster: 'Is master',
+          'isactivemaster 1': 'Master',
+          servername: 'Servername',
+          'servername 1': 'Servername',
+        },
+      },
+    },
+  ],
+};
+
+local alertsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'master_num_open_connections{job=~"$job", hbase_cluster=~"$hbase_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{hbase_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'alertlist',
+  title: 'Alerts',
+  description: 'Panel to report on the status of integration alerts.',
+  options: {
+    alertInstanceLabelFilter: '{job="integrations/apache-hbase"}',
```

Comment:
Looks like hardcoded value

Reply:
Currently yes. This is our first round of integrations implementing this new alert panel, so if there is an example implementation to go off of that would be very helpful.

Reply:
You can try using same dynamic filter you use for that dashboard:
`{job=~"${job:regex}", hbase_cluster=~"${hbase_cluster:regex}", instance=~"${instance:regex}"}`
note usage of 'advanced syntax'  though as it doesn't work otherwise.